examples/benchmarks.py

```diff
@@ -78,15 +79,14 @@ def bench_mem(test_size):
         if not i % 10000:
             # This is probably KB (i.e. 1000 bytes) when run on linux
             r = resource.getrusage(resource.RUSAGE_SELF).ru_maxrss - r_init
-            print('Memory usage for {:,} encrypted numbers = {:,} ({:.4f} per number)'.format(
-                i, r, i and r / i
-            ))
+            print('Memory usage for {:,} encrypted numbers = {:,} ({:.4f} per '
+                  'number)'.format(i, r, i and r / i))
         nums.append(pubkey.encrypt(random.random()))
 
-#bench_mem(1000000)  # NOTE: this will take a long time
+# bench_mem(1000000)  # NOTE: this will take a long time
```

Comment:
I guess, to give a demo of something much larger. That's why it is commented out.